t/config.t

```diff
@@ -224,4 +223,20 @@ subtest 'Validation of worker timeout' => sub {
     };
 };
 
+subtest 'parse ignored_failed_minion_jobs' => sub {
```

Comment:
It would be less effort to extend the existing subtest `Test configuration override from file` instead of creating a new, similarly structured subtest for just one field.

Reply:
Note that you're still "requesting changes" from a previous comment.

Reply:
fixed.